torch/csrc/jit/codegen/cuda/kernel_cache.cpp

```diff
@@ -185,6 +186,36 @@ at::DimVector inversePermutation(
 
 } // namespace
 
+size_t InputsCodeLookup::getCode(const at::ArrayRef<IValue>& inputs) {
```

Comment:
Please ignore the implementation here. This is a naive approach. I'm working on a trie to replace the stringstream.